.pre-commit-config.yaml

```diff
@@ -17,10 +17,10 @@ repos:
   - id: reorder-python-imports
 - repo: local
   hooks:
-  - id: lint-frontend
+  - id: lint-frontend-format
     name: Linting of JS, Vue, SCSS and CSS files on the frontend
     description: This hook handles all frontend linting for Kolibri
-    entry: yarn run lint-frontend
+    entry: yarn run lint-frontend:format
```

Comment:
maybe it's better to do this change in a different commit.

Reply:
Yeah. The problem was that pre-commit wouldn't let me commit when the configuration is changed. But I'll split it!